Assets/Prefabs/UI/Screens/Title Screen.prefab

```diff
@@ -1665,19 +1667,19 @@ MonoBehaviour:
   title_credits_exit: {fileID: 7008280155777404917}
   title_credits_text: {fileID: 6635772105199850637}
   title_credits_screen: {fileID: 7008280156646255309}
-  resolutions_settings_button: {fileID: 9106280308630234267}
+  resolutions_settings_button: {fileID: 0}
   resolutions_settings_exit: {fileID: 8432064966437982185}
   resolutions_settings_screen: {fileID: 8432064965012767701}
-  default_full_resolution_button: {fileID: 4323211303698117341}
-  higher_full_resolution_button: {fileID: 6395138741971958946}
-  highest_full_resolution_button: {fileID: 7912954396800630687}
-  lower_full_resolution_button: {fileID: 5169184802478309363}
-  lowest_full_resolution_button: {fileID: 1099097683826634511}
-  default_windowed_resolution_button: {fileID: 6047062747406663586}
-  higher_windowed_resolution_button: {fileID: 6051189693243452235}
-  highest_windowed_resolution_button: {fileID: 8039962976778249527}
-  lower_windowed_resolution_button: {fileID: 2014655415589967374}
-  lowest_windowed_resolution_button: {fileID: 4967393487091379957}
+  default_full_resolution_button: {fileID: 0}
```

Comment:
can you check that these aren't empty in Unity? i'm assuming they've been changed to modifications rather than saved to the Title Screen prefab. but i'm a little concerned they're all getting zeroed out